### DISCUSSION TITLE
pnpm monorepo modules not found when deploying with Vercel

### DISCUSSION THREAD

**Discussion by NettaB** [vercel/next.js#37501] [Help]
I'm working on a monorepo using pnpm workspace, where one of the apps is a NextJS app deployed with Vercel. Local build works as expected, but when building remote I get a `module not found` error for modules in the workspace (being imported with '@<workspace-name>/<package>`. These packages appear in `package.json`, and have previously been correctly imported into the project, but now for some reason cannot be used. 
The root repository defined in Vercel is the NextJS app, and it is configure to include outside sources.

Has anyone run into this? What is the most reliable way to consume files form within a monorepo?

**Comment by knowivate** [comment]
Have you found solution?
I am also facing similar type of issues.
Basically, I am using netlify to deploy my site. My site works perfect when I use it on local but when I deploy it on netlify It shows `module not found error` if I use `@repo/libname`

**Comment by zeJabun** [comment]
I had the same issue as you, I opted to use turborepo in combination with pnpm workspaces. I also made sure to remove any custom build commands in the Vercel project's `Build & Development Settings`.
